CONTRIBUTING.md

```diff
@@ -52,8 +92,9 @@ Every pull request automatically runs the continuous integration with every upda
 
 The continuous integration must be **successfully completed** with `All checks have passed` before proceeding with the approval process.
 
-### Schema Validation
-We use the sample programs from [schema-validation] repository in the continuous integration to validate the schemas against the [JSON], [XML] and [RDF] examples from the aas-spec repository.
+### IDTA-01001: Schema Validation
+We use the sample programs from [schema-validation] repository in the continuous integration to validate the 
```

Comment:
I understand why the word "program" is used here, still it sounds strange. Maybe "tools" or "executables"?